templates/snap-details.html

```diff
@@ -1,19 +1,40 @@
 {% extends "_layout.html" %}
 
-{% block title %}{{ snap_title }} — Linux software in the Snap Store{% endblock %}
-
-{% block meta %}
-  <meta property="og:title" content="{{ snap_title }} — Linux software in the Snap Store"/>
-  <meta property="og:url" content="https://snapcraft.io/{{ package_name }}"/>
-  <meta property="og:site_name" content="Snapcraft"/>
-  <meta property="og:type" content="product"/>
-
-  {% if summary %}<meta property="og:description" content="{{ summary }}"/>{% endif %}
-  {% if icon_url %}
-    <meta property="og:image" content="{{ icon_url }}"/>
-  {% else %}
-    <meta property="og:image" content="https://assets.ubuntu.com/v1/e635d1ef-snapcraft_green-red_hex.png" />
+{% block title %}{{ snap_title }} - Linux software in the Snap Store{% endblock %}
```

Comment:
Maybe we should rename all meta blocks to `meta_` to make a clearer distinction between them and content blocks?

Reply:
good idea :+1: